SocialCareCaseViewerApi/V1/Infrastructure/AllocationSet.cs

```diff
@@ -23,6 +23,15 @@ public class AllocationSet : IAuditEntity
         [MaxLength(62)]
         public int? TeamId { get; set; }
 
+        [Column("rag_rating")]
```

Comment:
there'll be merge conflicts here with the other branches. Identical changes so easy to resolve